cmd/modulegen/templates/x/client/cli/query_template.tmpl

```diff
@@ -0,0 +1,22 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/osmosis-labs/osmosis/osmoutils/osmocli"
+	"github.com/osmosis-labs/osmosis/v15/x/concentrated-liquidity/types/query"
+	"github.com/osmosis-labs/osmosis/v15/x/poolmanager/client/queryproto"
+	"github.com/osmosis-labs/osmosis/v15/x/poolmanager/types"
+)
+
+// GetQueryCmd returns the cli query commands for this module.
+func GetQueryCmd() *cobra.Command {
+	cmd := osmocli.QueryIndexCmd(types.ModuleName)
+
+	cmd.AddCommand(
+		osmocli.GetParams[*queryproto.ParamsRequest](
+			types.ModuleName, query.NewQueryClient),
```

Comment:
with the imports specified, I think it will always use `poolmanager` as module name? And query client from CL